.gitignore

```diff
@@ -3,4 +3,6 @@ bower_components
 .grunt
 styleguide/build
 public/css/*
+public/js/components/*.js
```

Comment:
Do you need to commit a README or something in the ``components`` directory so that it gets created on repo clones?